docs/Release-Process.md

```diff
@@ -5,44 +5,67 @@ Containers release
-- The kata-containers source branch that we use in the [podvm `Dockerfiles`](../src/cloud-api-adaptor/podvm/) and the
-[podvm workflows](../.github/workflows), by updating the `git.kata-containers.reference` value in [versions.yaml](../src/cloud-api-adaptor/versions.yaml) to
-the tag of the kata-containers release candidate.
-- The `kata-containers/src/runtime` go module that we include in the main `cloud-api-adaptor` [`go.mod`](../src/cloud-api-adaptor/go.mod) and the `csi-wrapper` [`go.mod`](../src/csi-wrapper/go.mod).
-This can be done by running
-    ```
-    go get github.com/kata-containers/kata-containers/src/runtime@<release candidate branch e.g. CCv0>
-    go mod tidy
-    ```
-in the [cloud-api-adaptor](../src/cloud-api-adaptor/) directory and [csi-wrapper](../src/csi-wrapper/) directory.
-
-- The attestation-agent that is built into the peer pod vm image, by updating the `git.guest-components.reference` value in [versions.yaml](../src/cloud-api-adaptor/versions.yaml)
-
-Please ensure podvm image build don't break:
-- build podvm image manually
-- or use the GHA [Publish pod VM Images](https://github.com/confidential-containers/cloud-api-adaptor/blob/main/.github/workflows/podvm_publish.yaml)
-
-These updates should be done in a PR that is merged triggering the [project images publish workflow](../.github/workflows/publish_images_on_push.yaml) to create a new container image in
-[`quay.io/confidential-containers/cloud-api-adaptor`](https://quay.io/repository/confidential-containers/cloud-api-adaptor?tab=tags) to use in testing.
+In the new Confidential Containers
+[release process](https://github.com/confidential-containers/community/blob/main/.github/ISSUE_TEMPLATE/release-check-list.md),
+the plan is to do a succession of component releases, rather than releasing
+all components at the same time. This means that the peer pods release process needs to slot into the correct place.
+The flow of releases should roughly be:
+1. The [guest-components release](https://github.com/confidential-containers/guest-components/releases) (or a pinned
+version is picked) and [trustee releases](https://github.com/confidential-containers/trustee/releases).
+    - This triggers [kata-containers](https://github.com/kata-containers/kata-containers) to update to these new versions in
+    [versions.yaml](https://github.com/kata-containers/kata-containers/blob/main/versions.yaml) under
+    `externals.coco-guest-components.version`, `externals.coco-trustee` and the `image-rs` crate in the agent's
+    [`Cargo.toml`](https://github.com/kata-containers/kata-containers/blob/main/src/agent/Cargo.toml).
+    - At this point it makes sense for us to stay in sync, by updating the guest-components and kbs that we use in peer pods,
+    by changing the `git.guest-components.reference` and `git.kbs.tag` values in [versions.yaml](../src/cloud-api-adaptor/versions.yaml).
+    We should also bump the kata agent to the latest commit
```

Comment:
Hi @stevenhorsman !

Should we update the agent's version in caa again after kata's release?

Reply:
That's mentioned in L25 & 26?

Reply:
True; never mind.